internal/comm/catchup_handler_test.go

```diff
@@ -0,0 +1,211 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package comm_test
```

Comment:
Maybe I miss something, but why do we need extra package for `comm_test`?

Reply:
It is just that I prefer block-box testing, not white-box. I never access non exported vars or methods in these tests.